src/org/parosproxy/paros/extension/report/ReportLastScan.java

```diff
@@ -63,16 +64,35 @@
     private static final String HTML_FILE_EXTENSION=".html";
     private static final String XML_FILE_EXTENSION=".xml";
     private static final String MD_FILE_EXTENSION=".md";
+    private static final String JSON_FILE_EXTENSION=".json";
     
-    public enum ReportType {HTML, XML, MD}
+    public enum ReportType {HTML, XML, MD, JSON}
 
     public ReportLastScan() {
     }
 
 
-    public File generate(String fileName, Model model, String xslFile) throws Exception {
+    public File generate(String fileName, Model model, ReportType reportType) throws Exception {
```

Comment:
This breaks binary compatibility, better leave/adapt the "old" method in case an add-on is using it (or, change this one to use the "old").